### DISCUSSION TITLE
Texts from SVG files not rendered

### DISCUSSION THREAD

**Discussion by andre2007** [AngusJohnson/Image32#108] [General]
If I understand correctly, Image32 should be able to render text, but the texts from this SVG file are not shown.

This is the expected output:

![image](https://github.com/user-attachments/assets/3a5bb49b-99ed-4522-ad62-4500d28dde24)

but in my FMX application it is rendered like this:

![image](https://github.com/user-attachments/assets/e720a5be-b88c-4455-aadc-aa8831b67f10)

This is the SVG file
[sample_chords.zip](https://github.com/user-attachments/files/17248665/sample_chords.zip)

Is there anything special with the SVG files produced by Verovio causing the texts are not rendered?
(I tested it with latest main of Image32)

**Comment by andersmelander** [comment]
See #59

**Comment by andre2007** [comment]
Thanks for the info, it is strange that it worked for you on Windows 64 by default.

My issue is on Windows64 and the font is a common one (`{font-family:Times,serif;}`).
I will try the recommendation from #59 maybe it solves my issue also on Windows 64.

**Comment by andersmelander** [comment]
> it worked for you

I'm just a lurker; I don't use Image32.

As far as I know the Image32 font manager doesn't automatically resolve and load fonts from the host system. You have to manually add a set of fonts to it and it will then use those to resolve what ever font request it get. There's probably an example that demonstrates how it's done.

**Comment by AngusJohnson** [comment]
> As far as I know the Image32 font manager doesn't automatically resolve and load fonts from the host system. You have to manually add a set of fonts to it and it will then use those to resolve what ever font request it get. 

Correct.

> There's probably an example that demonstrates how it's done.

Correct again 😁. See ./Examples/FMX_SVG.

**Comment by andre2007** [comment]
Thank you, it works fine now 😊
